src/test/java/com/sun/tools/xjc/addon/xew/XmlElementWrapperPluginTest.java

```diff
@@ -34,216 +52,223 @@
+	private static final String GENERATED_SOURCES_PREFIX = "target/test/generated_xsd_classes/";
 
-	private static final String	PREGENERATED_SOURCES_PREFIX	= "src/test/generated_resources/";
-	private static final String	GENERATED_SOURCES_PREFIX	= "target/test/generated_xsd_classes/";
-
-	private static final Log	logger						= LogFactory.getLog(XmlElementWrapperPluginTest.class);
+	private static final Log logger = LogFactory.getLog(XmlElementWrapperPluginTest.class);
 
 	@Test
-	public void testUsage() throws Exception {
-		assertNotNull(new XmlElementWrapperPlugin().getUsage());
+	public void testUsage() {
+		assertThat(new XmlElementWrapperPlugin().getUsage()).isNotNull();
 	}
 
-	@Test(expected = BadCommandLineException.class)
-	public void testUnknownOption() throws Exception {
-		runTest("different-namespaces", new String[] { "-Xxew:unknown" }, false);
+	@Test
+	public void testUnknownOption() {
+		assertThatThrownBy(() -> runTest("different-namespaces", singletonList("-Xxew:unknown"), false, emptyList()))
+				.isInstanceOf(BadCommandLineException.class);
 	}
 
-	@Test(expected = IllegalArgumentException.class)
-	public void testInvalidInstantiationMode() throws Exception {
-		runTest("element-list-extended", new String[] { "-Xxew:instantiate invalid" }, false);
+	@Test
+	public void testInvalidInstantiationMode() {
+		assertThatThrownBy(() -> runTest("element-list-extended", singletonList("-Xxew:instantiate invalid"), false, emptyList()))
+				.isInstanceOf(IllegalArgumentException.class);
 	}
 
-	@Test(expected = BadCommandLineException.class)
-	public void testInvalidControlFile() throws Exception {
-		runTest("element-list-extended", new String[] { "-Xxew:control invalid" }, false);
+	@Test
+	public void testInvalidControlFile() {
+		assertThatThrownBy(() -> runTest("element-list-extended", singletonList("-Xxew:control invalid"), false, emptyList()))
+				.isInstanceOf(BadCommandLineException.class);
 	}
 
-	@Test(expected = BadCommandLineException.class)
-	public void testInvalidCollectionClass() throws Exception {
-		runTest("element-list-extended", new String[] { "-Xxew:collection badvalue" }, false);
+	@Test
+	public void testInvalidCollectionClass() {
+		assertThatThrownBy(() -> runTest("element-list-extended", singletonList("-Xxew:collection badvalue"), false, emptyList()))
+				.isInstanceOf(BadCommandLineException.class);
 	}
 
-	@Test(expected = IllegalArgumentException.class)
-	public void testInvalidCustomization() throws Exception {
-		runTest("element-with-invalid-customization", null, false);
+	@Test
+	public void testInvalidCustomization() {
+		assertThatThrownBy(() -> runTest("element-with-invalid-customization", emptyList(), false, emptyList()))
+				.isInstanceOf(IllegalArgumentException.class);
 	}
 
-	/**
-	 * This test works reliably on Java7 but produces different results from run to run on Java8.
-	 */
 	@Test
+	@Disabled("This test works reliably on Java7 but produces different results from run to run on Java8.")
```

Comment:
Disabled this test because it is not deterministic due to the mentioned behavior